1.51.4 fails to compile: error: loop variable '[target, setId]' creates a copy from type 'const value_type' (aka 'const pair<const unsigned int, unsigned int>')

⚠️ **Issues not using this template will be systematically closed.**

**Describe the bug**
```
/wrkdirs/usr/ports/graphics/filament/work/filament-1.51.4/filament/backend/src/vulkan/spirv/VulkanSpirvUtils.cpp:186:21: error: loop variable '[target, setId]' creates a copy from type 'const value_type' (aka 'const pair<const unsigned int, unsigned int>') [-Werror,-Wrange-loop-construct]
    for (auto const [target, setId]: targetToSet) {
                    ^
/wrkdirs/usr/ports/graphics/filament/work/filament-1.51.4/filament/backend/src/vulkan/spirv/VulkanSpirvUtils.cpp:186:10: note: use reference type 'const value_type &' (aka 'const pair<const unsigned int, unsigned int> &') to prevent copying
    for (auto const [target, setId]: targetToSet) {
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                    &
1 error generated.
```

**To Reproduce**
Regular build.

**Desktop (please complete the following information):**
 - OS: FreeBSD 14.0
 - Compiler: clang-18

--- a/filament/backend/src/vulkan/spirv/VulkanSpirvUtils.cpp
+++ b/filament/backend/src/vulkan/spirv/VulkanSpirvUtils.cpp
@@ -183,7 +183,7 @@ std::tuple<uint32_t,uint32_t, uint32_t> getProgramBindings(Program::ShaderBlob c
     constexpr uint32_t IATTACHMENT = 2;
     uint32_t ubo = 0, sampler = 0, inputAttachment = 0;
 
-    for (auto const [target, setId]: targetToSet) {
+    for (auto const& [target, setId]: targetToSet) {
         uint32_t const binding = targetToBinding[target];
         assert_invariant(binding < 32);
         switch(setId) {

